web/src/views/StatKeeper/bookkeeper.ts

```diff
@@ -49,11 +45,11 @@ export class Bookkeeper {
     week: number,
     homeTeam: { id: number; name: string },
     awayTeam: { id: number; name: string },
-    homeRoster: string[],
-    awayRoster: string[]
+    homeRoster: StoredPlayer[],
+    awayRoster: StoredPlayer[]
```

Comment:
this constructor is awkward. It should take a proper team type with roster instead of both separate